Gemfile

```diff
@@ -0,0 +1,60 @@
+source "https://rubygems.org"
+git_source(:github){|repo| "https://github.com/#{repo}.git"}
+
+ruby "2.7.0"
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem "rails", "~> 6.0.3", ">= 6.0.3.2"
+# Use sqlite3 as the database for Active Record
+gem "sqlite3", "~> 1.4"
```

Comment:
use gem mysql2 instead of sqlite3

Reply:
after i use mysql2 instead of sqlite3, the adapter is still kept as sqlite3... Should i change it?
![image](https://user-images.githubusercontent.com/68467813/89927225-9b4cc380-dc38-11ea-84b8-66b33fd01520.png)

Reply:
you neeed change config
`default: &default
  adapter: mysql2
  encoding: utf8
  pool: 5
  username: root
  password: root
  socket: /var/run/mysqld/mysqld.sock

development:
  <<: *default
  database: sample_development

test:
  <<: *default
  database: sample_test
`

Reply:
however after i ignored the database.yml, the file is cant be found... currently, only database.yml.example exist